lib/plugins/aws/provider.js

```diff
@@ -480,6 +480,25 @@ class AwsProvider {
             required: ['Fn::Sub'],
             additionalProperties: false,
           },
+          awsCfToJsonString: {
+            type: 'object',
+            properties: {
+              'Fn::ToJsonString': {
+                oneOf: [
+                  { type: 'object' },
+                  { type: 'array' },
+                  { $ref: '#/definitions/awsCfFunction' },
+                  { $ref: '#/definitions/awsCfFindInMap' },
+                  { $ref: '#/definitions/awsCfGetAZs' },
+                  { $ref: '#/definitions/awsCfIf' },
+                  { $ref: '#/definitions/awsCfSelect' },
+                  { $ref: '#/definitions/awsCfSplit' },
```

Comment:
Let's remove those definitions (it's fine to simply allow `object` and let follow CF validator with deep validation)